kivy/core/camera/camera_avfoundation.pyx

```diff
@@ -62,7 +62,7 @@ class CameraAVFoundation(CameraBase):
         self._resolution = (width, height)
 
         if self._texture is None or self._texture.size != self._resolution:
-            self._texture = Texture.create(self._resolution)
+            self._texture = Texture.create(self._resolution, colorfmt='bgra')
```

Comment:
Looks very easy to add it, however, the pixel format is different than the default. Is that also the case for OSX?
I think this part may break OSX, so it should be per platform